projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.html

```diff
@@ -3,9 +3,11 @@
     <div class="properties">
 
         <!-- list of properties -->
-        <div *ngFor="let prop of propArray; let last = last" [class.border-bottom]="!last">
+        <div *ngFor="let prop of propArray; let last = last">
             <!-- show property; all in case of showAll === true or only the ones with prop.values -->
-            <div class="property" *ngIf="!prop.propDef['isLinkProperty'] && showAllProps || ( prop.values && prop.values.length > 0 )">
+            <div *ngIf="!prop.propDef['isLinkProperty'] && showAllProps || ( prop.values && prop.values.length > 0 )"
+                [class.border-bottom]="prop.values"
```

Comment:
You should also keep `!last` &rarr; `[class.border-bottom]="prop.values && !last"`
Because last property does not need a bottom border.

Reply:
This I wasn't sure, should ask ;). Done with 3660790.